packages/odyssey-react-mui/src/Select.tsx

```diff
@@ -433,6 +434,7 @@ const Select = <
           {...inputValues}
           aria-describedby={ariaDescribedBy}
           aria-errormessage={errorMessageElementId}
+          displayEmpty
```

Comment:
Good catch here. I could have sworn we added this prop previously.

Reply:
We did have this [here previously!](https://github.com/okta/odyssey/commit/e4507f66ace98b09dea8e331ca11d615e2a1e09a#diff-1e4d080b7bbf21c0a69512d075fad69a8ea134c102a54faefc5daae234a57eeaL328) I knew I wasn't crazy 😆

Reply:
May have gotten removed in error